jvm/core/main/scala/com/webank/eggroll/core/resourcemanager/metadata/DbModel.scala

```diff
@@ -16,3 +19,13 @@ case class DbStorePartition(storePartitionId: Long = -1,
                             nodeId: Long = -1,
                             partitionId: Int = 0,
                             status: String = StringConstants.EMPTY)
+
+case class DbServerNode(id: Long = -1,
+                        name: String = StringConstants.EMPTY,
+                        clusterId: Long = 0,
+                        endpoint: ErEndpoint = ErEndpoint(host = StringConstants.EMPTY, port = -1),
+                        nodeType: String = StringConstants.EMPTY,
+                        status: String = StringConstants.EMPTY,
+                        lastHeartbeatAt: Date = new Date(),
+                        createdAt: Date = new Date(),
+                        updatedAt: Date = new Date())
```

Comment:
Better use a special default value for these 3 fields, e.g. 1970-01-01 00:00:00

Reply:
No dependency on them at the current version. Approve for now, but they should be modified in the next push.